### PR TITLE
Updating geolocation documentation (Task 47985255)

### PR DESCRIPTION
**Pull request by vwehren**
Updates Geolocation privacy documentation as per [Task 47985255]Needed by Stable: Edge Public Privacy Whitepaper: Geolocation services. This update more clearly reflects that Edge uses a Microsoft-provided location service when the user gives the site permission, but the device's native location services are turned off/unavailable, in line with what all browsers currently do on the Windows platform. 

Also, instead of describing how to turn on/off the Windows location services, the article now links to the Windows documentation, given the differences in Windows 10 vs. 11 or future versions of Windows. 

Rendered article section for review:
https://review.learn.microsoft.com/microsoft-edge/privacy-whitepaper/?branch=pr-en-us-2957#geolocation
[Public GitHub preview](https://github.com/MicrosoftDocs/edge-developer/blob/user/vwehren/docs/privacy_wp_edits/microsoft-edge/privacy-whitepaper/index.md#geolocation)

[Live article section](https://learn.microsoft.com/microsoft-edge/privacy-whitepaper/#geolocation)

AB#48020686

### CODE REVIEW

File: microsoft-edge/privacy-whitepaper/index.md [hunk 1]
```diff
@@ -390,21 +390,20 @@ To stop Microsoft Edge from finding related matches in **Find on page**:
...
+When the site you are on has access to your location, Microsoft Edge shows a location icon in the address bar. You can select this icon to get more information and to manage the location access permissions for this site.
 
-If you turn off the **Allow access to location on this device** and **Allow apps to access your location** settings, some sites may still estimate your location using other technologies (such as Bluetooth, WiFi, cellular modem, or IP address) with varying degrees of accuracy. With Windows location settings disabled, precise location experiences that you allow in Microsoft Edge may also be inaccurate. For more information about Windows location settings, see [Windows location service and privacy](https://support.microsoft.com/help/4468240).
+![Location](./index-images/geolocation2.png)
 
-Microsoft Edge doesn't store your geolocation coordinates. When making requests to the Windows location service, Microsoft Edge generates a new random ID for each request.
+Microsoft Edge doesn't collect or store the location data that is shared with the site and generates a new random identifier for each request.
```
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:404]:
```suggestion
Microsoft Edge doesn't collect or store the location data that is shared with the site, and generates a new random identifier for each request.
```
or, two separate sentences b/c seems two distinct points

**Review by mikehoffms — CHANGES_REQUESTED** [review]
Writer/Editor review

**Review by apryzgoda — APPROVED** [review]
Edge Privacy Approved

**Review by mikehoffms — APPROVED** [review]
Approved the PR per Writer/Editor pass for Docs team.

**Pull request merged**